src/doc/man/cargo-tree.adoc

```diff
@@ -0,0 +1,193 @@
+----
+myproject v0.1.0 (/myproject)
+└── rand v0.7.3
+    ├── getrandom v0.1.14
+    │   ├── cfg-if v0.1.10
+    │   └── libc v0.2.68
+    ├── libc v0.2.68 (*)
+    ├── rand_chacha v0.2.2
+    │   ├── ppv-lite86 v0.2.6
+    │   └── rand_core v0.5.1
+    │       └── getrandom v0.1.14 (*)
+    └── rand_core v0.5.1 (*)
+[build-dependencies]
+└── cc v1.0.50
+----
+
+Packages marked with `(*)` have been "de-duplicated". The dependencies for the
+package have already been shown elswhere in the graph, and so are not
+repeated. Use the `--no-dedupe` option to repeat the duplicates.
+
+== OPTIONS
+
+=== Tree Options
+
+*-i*::
+*--invert*::
+    Invert the tree. Typically this flag is used with the `-p` flag to show
+    the dependencies for a specific package.
+
+*--no-dedupe*::
+    Do not de-duplicate repeated dependencies. Usually, when a package has
+    already displayed its dependencies, further occurrences will not
+    re-display its dependencies, and will include a `(*)` to indicate it has
+    already been shown. This flag will cause those duplicates to be repeated.
+
+*-d*::
+*--duplicates*::
+    Show only dependencies which come in multiple versions (implies
+    `--invert`). When used with the `-p` flag, only shows duplicates within
+    the subtree of the given package.
++
+It can be beneficial for build times and executable sizes to avoid building
+that same package multiple times. This flag can help identify the offending
+packages. You can then investigate if the package that depends on the
+duplicate with the older version can be updated to the newer version so that
+only one instance is built.
+
+*--no-dev-dependencies*::
+    Do not include dev-dependencies.
+
+*--target* _TRIPLE_::
+    Filter dependencies matching the given target-triple.
+    The default is the host platform.
+
+*--no-filter-targets*::
+    Show dependencies for all target platforms. Cannot be specified with
+    `--target`.
+
+*--graph-features*::
+    Runs in a special mode where features are included as individual nodes.
+    This is intended to be used to help explain why a feature is enabled on
+    any particular package. It is recommended to use with the `-p` and `-i`
+    flags to show how the features flow into the package. See the examples
+    below for more detail.
+
+=== Tree Formatting Options
+
+*--charset* _CHARSET_::
+    Chooses the character set to use for the tree. Valid values are "utf8" or
+    "ascii". Default is "utf8".
+
+*-f* _FORMAT_::
+*--format* _FORMAT_::
+    Set the format string for each package. The default is "{p}".
++
+This is an arbitrary string which will be used to display each package. The following
+strings will be replaced with the corresponding value:
++
+- `{p}` — The package name.
+- `{l}` — The package license.
+- `{r}` — The package repository URL.
+- `{f}` — Comma-separated list of package features that are enabled.
+
+*--no-indent*::
+    Display the dependencies as a list (rather than a tree).
+
+*--prefix-depth*::
```

Comment:
I'd be in favor of consolidating options where possible, like your suggestion here to merge `--prefix-depth` and `--no-indent`

Reply:
Consolidated into `--prefix`.